deeppavlov/datasets/robust_dataloader.py

```diff
@@ -0,0 +1,197 @@
+import codecs
+import os
+from glob import glob
+
+from nltk import sent_tokenize, word_tokenize
+from six.moves import cPickle
+import numpy as np
+from tqdm import tqdm
+from collections import Counter
+
+
+class TextLoader:
```

Comment:
3. Add a docstring.

Reply:
4. The class is not wrapped into DeepPavlov abstractions. Looks like this class should be inherited from `DatasetReader` class and should have a proper name (sth like "BlaBlaReader").
Also there is a separate base class `DefaultVocabulary` which should be used here. If `DefaultVocabulary` is not good for this usecase, a separate Vocabulary class should be written.